### PR TITLE
fix: home navigation layout

### PR DESCRIPTION
**Pull request by zevik7**

### CODE REVIEW

**Comment by trucho1001** [comment]
omh :)) fix

**Comment by zevik7** [comment]
> omh :)) fix

File: src/pages/BicycleDetail/style.less [hunk 1]
```diff
@@ -119,3 +119,25 @@
     }
   }
 }
+
```
Comment (nguyendangnfq) [inline src/pages/BicycleDetail/style.less:122]: Remove completely it @zevik7

File: src/pages/NoMatch/index.js [hunk 2]
```diff
@@ -13,9 +14,9 @@ const NoMatch = () => {
         style={{
           paddingTop: '128px',
         }}
-        status="404"
         title="404"
         subTitle="Sorry, the page you visited does not exist."
+        icon={<img src={NotFoundIcon} />}
```
Comment (nguyendangnfq) [inline src/pages/NoMatch/index.js:19]: this is amazing GIF LOL :)))

**Review by nguyendangnfq — CHANGES_REQUESTED** [review]

**Review by nguyendangnfq — APPROVED** [review]

**Pull request merged**